### ISSUE TITLE
How do I display the captcha icon only on certain pages (VUE reCAPTCHA-v3)?

### ISSUE THREAD

**Issue by positivethinking639** [AurityLab/recaptcha-v3#105]
I use this package : https://www.npmjs.com/package/vue-recaptcha-v3. It get from this github

I add on my main.js : 

    import { VueReCaptcha } from 'vue-recaptcha-v3'
    
    Vue.use(VueReCaptcha, { siteKey: 'xxxxxxx' })

I add this code :

    await this.$recaptcha('login').then((token) => {
        recaptcha = token
    })

to my component to get token from google recapchta

My problem is the captcha icon in the lower right corner appears on all pages

[![enter image description here][1]][1]

I want it to only appear in certain components

Maybe I must to change this : `    Vue.use(VueReCaptcha, { siteKey: 'xxxxxxxxxxxxxxxxx' })`. Seems it still mounting to `Vue.use`. I want to mount to a certain component instead of vue root instance 

How can I solve this problem?

  [1]: https://i.stack.imgur.com/ufp8S.png

**Comment by WipeAir** [comment]
Hi,
ReCaptcha-v3 is intended to be loaded on all pages. If you just want to get rid of the badge you can use the [`autoHideBadge`](https://github.com/AurityLab/recaptcha-v3#loader-options) option.

**Comment by positivethinking639** [comment]
yeah but that hides the badge. I really want to hide the badge. but I want
the badge to only appear 1 page, namely on the registration page.

**Comment by WipeAir** [comment]
There are [`hideBadge()`](https://github.com/AurityLab/recaptcha-v3/blob/master/src/ReCaptchaInstance.ts#L43) and [`showBadge()`](https://github.com/AurityLab/recaptcha-v3/blob/master/src/ReCaptchaInstance.ts#L54) methods on the [instance](https://github.com/AurityLab/recaptcha-v3/blob/master/src/ReCaptchaInstance.ts) (Will be returned by `load('<site key>')`).
I'd suggest you to just load ReCaptcha on your root instance instead of using some kind of workaround for this.

**Comment by positivethinking639** [comment]
I install this package : https://www.npmjs.com/package/vue-recaptcha-v3.
Because it's vue and it's google recaptcha version 3. is it the same as
yours or not?

**Comment by WipeAir** [comment]
`vue-recaptcha-v3` is based on this package. But there is currently no way to access the ReCaptcha Instance.

**Comment by positivethinking639** [comment]
then what is the solution to solve my problem? Do I have to uninstall this
package and install your package? or is there a way to overcome that with
this package? I just want the badge to appear on 1 page. that is it

**Comment by WipeAir** [comment]
Hey, sorry for the delayed answer. 
As `vue-recpatcha-v3` is just a wrapper package you can use this package if you want to. 
Anyways, I'll publish a new version of `vue-recaptcha-v3` which will expose the API of this package within the next days.
